Source/PLCrashSignalHandler.mm

```diff
@@ -419,7 +419,7 @@ - (BOOL) registerHandlerForSignal: (int) signo
         .callback = callback,
         .context = context
     };
-    shared_handler_context.callbacks.nasync_prepend(reg);
+    shared_handler_context.callbacks.nasync_append(reg);
```

Comment:
It's supposed that last registered handler will be called first.

Also, as mentioned in the comment for this method - forwarding to next callback in this chain is optional:
https://github.com/microsoft/plcrashreporter/blob/d4e2df216e43a2ba0fa747aaad155edff8652683/Source/PLCrashSignalHandler.mm#L399-L401